framework/axSGreaseSkeleton.js

```diff
@@ -11,6 +11,59 @@
 
 /*** Functions for common tweaks. ***/
 
+/**
+ * Adds text to the given live region, and clears it a second later so it's no
+ * longer in the virtual buffer.
```

Comment:
nit: This isn't specific to virtual buffers; e.g. it impacts VoiceOver just as much.

```suggestion
 * longer perceivable.
```